content/docs/references/contributing.md

```diff
@@ -92,3 +92,9 @@ Ask the following questions to yourself whenever you find that you are unsure wh
 ### Need for judicial application
 
 It is important to reiterate that the decision will involve discrete judgment and possibly, discussion. Use the above points and the flowchart as a reference, but in the end, apply your judgment. We are flexible in changing our stance wherever it makes sense. Create an issue, open a PR, or just send a message on Slack to talk about your content and where it can go in the engineering handbook.
+
+### Markdown: Our Choice for Documentation
```

Comment:
```suggestion
## Markdown: Our Choice for Documentation
```

Right now, this heading is under the section "How to decide where to place content". It doesn't make sense there.

Reply:
Fixed